### PR TITLE
Minor fixes in stream quickstart

### PR DESCRIPTION
**Pull request by aruediger**
WDYT?

### CODE REVIEW

**Comment by ktoso** [comment]
Looks good, LGTM.
Needs to be applied to `/java/stream-quickstart.rst` too.

**Comment by 2beaucoup** [comment]
Done. Also streamlined the java code a bit while at it.

**Comment by ktoso** [comment]
OK TO TEST

File: akka-docs-dev/rst/java/stream-quickstart.rst [hunk 1]
```diff
@@ -154,29 +154,29 @@ So far we've been only processing data using Flows and consuming it into some ki
...
 First, let's write such an element counter using ``Flow.of(Class)`` and ``Sink.fold`` to see how the types look like:
 
 .. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java#tweets-fold-count
 
-First we prepare a reusable ``Flow`` that will change each incoming tweet into an integer of value ``1``.
-We combine all values of the transformed stream using ``Sink.fold`` will sum all ``Integer`` elements of the stream
-and make its result available as a ``Future<Integer>``. Next we connect the ``tweets`` stream though a ``map`` step which
-converts each tweet into the number ``1``, finally we connect the flow using ``toMat`` the previously prepared Sink.
+First we prepare a reusable ``Flow`` that will change each incoming tweet into an integer of value ``1``. We'll use this in
+order to combine those with a ``Sink.fold`` that will sum all ``Int`` elements of the stream and make its result available as
```
Comment (ktoso) [inline akka-docs-dev/rst/java/stream-quickstart.rst]: `Integer`, because java :)

Comment (2beaucoup) [inline akka-docs-dev/rst/java/stream-quickstart.rst]: Damn! One always slips through... Fixed.

Comment (ktoso) [inline akka-docs-dev/rst/java/stream-quickstart.rst]: :-) Thanks

**Comment by ktoso** [comment]
Thanks!

**Pull request merged**